src/webpush.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+const _ = require("lodash");
+const fs = require("fs");
+const path = require("path");
+const WebPushAPI = require("web-push");
+const Helper = require("./helper");
+
+class WebPush {
+	constructor() {
+		const vapidPath = path.join(Helper.HOME, "vapid.json");
+
+		if (fs.existsSync(vapidPath)) {
+			const data = fs.readFileSync(vapidPath, "utf-8");
+			const parsedData = JSON.parse(data);
+
+			if (typeof parsedData.publicKey === "string" && typeof parsedData.privateKey === "string") {
+				this.vapidKeys = {
+					publicKey: parsedData.publicKey,
+					privateKey: parsedData.privateKey,
+				};
+			}
+		}
+
+		if (!this.vapidKeys) {
```

Comment:
Yeah, that much I guessed, but it's a bit of a weird name and seems odd. I'm not saying you should do anything about it. Just braindumping, really.